library/Makefile

```diff
@@ -56,16 +56,16 @@ OBJS_CRYPTO=	aes.o		aesni.o		arc4.o		\
 		ecjpake.o	ecp.o				\
 		ecp_curves.o	entropy.o	entropy_poll.o	\
 		error.o		gcm.o		havege.o	\
-		hmac_drbg.o	md.o		md2.o		\
-		md4.o		md5.o		md_wrap.o	\
-		memory_buffer_alloc.o		oid.o		\
-		padlock.o	pem.o		pk.o		\
-		pk_wrap.o	pkcs12.o	pkcs5.o		\
-		pkparse.o	pkwrite.o	platform.o	\
-		platform_util.o	ripemd160.o	rsa_internal.o	\
-		rsa.o 		sha1.o		sha256.o	\
-		sha512.o	threading.o	timing.o	\
-		version.o	version_features.o		\
+		hmac_drbg.o	md.o        md2.o       \
+		md4.o		md5.o		md_wrap.o    \
```

Comment:
The alignment here doesn't look quite right. Please revise.

Reply:
I agree it doesn't look right, but on my editor it looks right..

Reply:
Please consider updating your editor configuration. I think a tab width of 8 is what you'd need to configure.

Reply:
@RonEld The alignment is wrong. If it looks right in your editor then you've configured it for some non-standard tab size. The standard tab size is 8. The other lines use tabs. Please use tabs for consistency (this is a makefile).

Reply:
That's fine, I used spaces instead of tabs in this place as well